tests/testsuite/doc.rs

```diff
@@ -513,16 +511,14 @@ fn doc_lib_bin_same_name_documents_named_bin_when_requested() {
         // The checking/documenting lines are sometimes swapped since they run
         // concurrently.
         .with_stderr_data(str![[r#"
-[WARNING] output filename collision.
```

Comment:
Could you adjust this so the content lines up for diffing?

Reply:
Done